packages/core-toggle/core-toggle.js

```diff
@@ -26,12 +26,7 @@ export default function toggle (buttons, open) {
 
 document.addEventListener('keydown', (event) => {
```

Comment:
Sorry, forgot this in last review 🙈  please use `addEvent(UUID, 'keydown')` to make sure the event is not bound several times when multiple instances is on same page 👍

Reply:
Ah nice! Fixed!